### ISSUE TITLE
Found 'path here' instead of resultsPath in cli

### ISSUE THREAD

**Issue by mixco** [tnicola/cypress-parallel#147]
Whenever I run the cypress-parallel I used to get an error
```
[Error: ENOENT: no such file or directory, unlink 'C:\\my-frontend-test\path here\cypress_e2e_specs_my.spec.cy.js.json'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'unlink',
  path: 'C:\\my-frontend-test\\path here\\cypress_e2e_specs_my.spec.cy.js.json'
```

I found in cli.js, line 25:
`fs.unlink(path.join('path here', file), err => { if (err) console.log(err); }); `
It seems like this '_path here_' was more like a TODO reminder, because when I changed it to
`fs.unlink(path.join(resultsPath, file), err => { if (err) console.log(err); }); `
no error was reported.

**Comment by dmitry** [comment]
@tnicola please update code, as unlink doesn't work because of that :)

**Comment by tnicola** [comment]
Thanks!This should be fixed in 0.11.0
